workloads/network-perf/run_network_compare.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -x
+
+datasource="elasticsearch"
+tool="uperf"
+function="compare"
+base_uuid=$1
```

Comment:
Minor nit but if you know the total number of arguments to expect why not just check for that instead of multiple if's?

Reply:
Smart! Too many after thoughts, thanks!